src/components/checkboxes/template.test.js

```diff
@@ -255,31 +255,82 @@ describe('Checkboxes', () => {
     })
   })
 
-  it('render conditional', () => {
-    const $ = render('checkboxes', {
-      name: 'example-conditional',
-      items: [
-        {
-          value: 'yes',
-          text: 'Yes'
-        },
-        {
-          value: 'no',
-          text: 'No',
-          checked: true,
-          conditional: {
-            html: 'Conditional content'
+  describe('render conditionals', () => {
+    it('hidden by default when not checked', () => {
+      const $ = render('checkboxes', {
+        name: 'example-conditional',
+        items: [
+          {
+            value: 'yes',
+            text: 'Yes',
+            conditional: {
+              html: 'Conditional content that should be hidden'
+            }
+          },
+          {
+            value: 'no',
+            text: 'No'
           }
-        }
-      ]
+        ]
+      })
+
+      const $component = $('.govuk-checkboxes')
+
+      const $firstConditional = $component.find('.govuk-checkboxes__conditional').first()
+      expect($firstConditional.html()).toContain('Conditional content that should be hidden')
```

Comment:
It'd be good to split this out into its own test and just have the one assertion per test.